src/watchdog/GOPATH/src/github.com/microsoft/watchdog/Gopkg.toml

```diff
@@ -0,0 +1,20 @@
+[prune]
+  go-tests = true
+  unused-packages = true
+
+[[constraint]]
+  name = "k8s.io/client-go"
+  version = "kubernetes-1.14.2"
+
+
+[[constraint]]
+  name = "github.com/prometheus/client_golang"
+  version = "1.1.0"
+
+[[constraint]]
+  name = "github.com/stretchr/testify"
+  version = "1.4.0"
+
+[[constraint]]
+  name = "github.com/microsoft/frameworkcontroller"
+  version = "0.5.1"
```

Comment:
Seems you do not have to depend on frameworkcontroller?

Reply:
I want to list frameworks here: https://github.com/microsoft/pai/blob/218d27517f92bc556fb283b3a8ecfeb7f3a401ff/src/watchdog/GOPATH/src/github.com/microsoft/watchdog/pkg/watchdog/k8s_client.go#L98

For GC, need to known if framework exist, then decide whether to delete the priorityClass or secret. I'm not sure if there is a better way.

Reply:
Let's try to use dynamic client as k8s GC controller

Reply:
Thanks, change it to dynamic client.